flyway-core/pom.xml

```diff
@@ -140,8 +140,8 @@
                         <Import-Package>
                             javax.sql,
                             org.apache.commons.logging;version="[1.1,2)";resolution:=optional,
-                            org.apache.logging.log4j;version="[2.15,3)";resolution:=optional,
-                            org.apache.logging.log4j.util;version="[2.15,3)";resolution:=optional,
+                            org.apache.logging.log4j;version="[2.16,0)";resolution:=optional,
```

Comment:
The original code says between versions 2.15 and 3. This change would say versions 2.16 to 0.